src/zap/zapheaders.h

```diff
@@ -249,7 +249,11 @@ class ZapDebugDirectory : public ZapNode
 
     virtual DWORD GetSize()
     {
+#if defined(NO_NGENPDB)
+        return sizeof(IMAGE_DEBUG_DIRECTORY) * m_nDebugDirectory;
+#else
```

Comment:
Can you add a comment explaining why we have a off by one difference here when computing the size?